clientd3d/d3drender.c

```diff
@@ -159,6 +159,8 @@ extern BYTE				light_rows[MAXY/2+1];      // Strength of light as function of sc
 extern ViewElement		ViewElements[];
 extern HDC				gBitsDC;
 
+D3DMATRIX view, mat, rot, trans, proj;

```

Comment:
View and proj are included in one of the structs; do we also need them as global variables, or can they be static now?